src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -318,10 +345,7 @@ class ReviewCollapsibleChapter extends React.Component {
     } = props;
     const ChapterDescription = chapterFormConfig.reviewDescription;
     return (
-      <div
-        className="usa-accordion-content schemaform-chapter-accordion-content"
-        aria-hidden="false"
-      >
+      <div data-testid="accordion-item-content">
```

Comment:
I'd like to avoid using `data-testid` unless it is required. Are we able to test this without?
My reasoning comes from the testing-library guiding principles, we should aim to test how users interact with the code. `getByTestId` uses the `data-testid` hook to locate the exact spot in the page, which is not how a user would interact with the same div. 

For reference: React Testing Query Priority (https://testing-library.com/docs/queries/about#priority)

Reply:
:) I generally agree! But here we don't really have anything useful to query for, and since this is only used when we're testing that we're _not_ rendering anything inside this container element I felt it was a fair concession to make. But I'm always happy to consider a different viewpoint if you have something to propose!

Reply:
Thanks! I'm fine with it provided we have cause to use it, I just want to make sure we've thought about alternatives and not used it as default (cause it is so much easier to write tests that query test ids)

Reply:
`schemaform-chapter-accordion-content` is still used a class [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/pre-need/components/CollapsiblePanel.jsx#L36), which is outside the scope of my work on this ticket so I don't want to touch it.

`usa-accordion-content` is also still used in a number of other places around the site, so those tests and styles will still be relevant for them:

```
src/applications/discharge-wizard/components/gpMinorComponents/AdditionalInstructions.jsx
src/applications/discharge-wizard/components/InstructionsPage.jsx
src/applications/proxy-rewrite/partials/header.js
src/applications/post-911-gib-status/components/EnrollmentPeriod.jsx
src/applications/edu-benefits/components/ConfirmationPageContent.jsx
src/applications/pre-need/components/CollapsiblePanel.jsx
src/applications/gi/components/AccordionItem.jsx
src/applications/gi/components/SearchAccordion.jsx
src/platform/site-wide/va-footer/components/MobileLinks.jsx
src/platform/site-wide/va-footer/components/LanguageSupport.jsx
src/platform/landing-pages/dev-template.ejs
```